src/baldr/CMakeLists.txt

```diff
@@ -1,24 +1,31 @@
 ## Process the IANA Time Zone Database files
 set(tz_db_files africa antarctica asia australasia backward etcetera europe
-  pacificnew northamerica southamerica systemv leapseconds)
+  northamerica southamerica leapseconds)
+
+# Process the leap seconds file (it's in .gitignore of the tz repo)
+add_custom_command(OUTPUT ${VALHALLA_SOURCE_DIR}/third_party/tz/leapseconds
+COMMAND make leapseconds
```

Comment:
I'm trying to build on Windows, following the instructions from the docs: https://valhalla.github.io/valhalla/building/#building-with-vcpkg-any-platform

However, it appear to stop on this. I think it's because of `make` and `awk` not being on my system. Are there additional pre-requisites to build on Windows?